detection_rules/schemas/__init__.py

```diff
@@ -197,6 +197,12 @@ def migrate_to_8_2(version: Version, api_contents: dict) -> dict:
     return strip_additional_properties(version, api_contents)
 
 
+@migrate("8.3")
+def migrate_to_8_3(version: Version, api_contents: dict) -> dict:
+    """Default migration for 8.3."""
+    return strip_additional_properties(version, api_contents)
```

Comment:
We will need to update this with the changes to the rule schema (in a separate PR, as discussed)